srcpkgs/tailscale/template

```diff
@@ -13,7 +13,7 @@ maintainer="Noel Cower <ncower@nil.dev>"
 license="BSD-3-Clause"
 homepage="https://tailscale.com"
 distfiles="https://github.com/tailscale/tailscale/archive/v${version}.tar.gz"
-checksum=@6b48993cee06397e22fd70c71d93ec9b2431cdb32a7e5cb37b515f7d34cbadb7
+checksum=@687833114a7294dc98c658502a4e4ea997bd467ad42c901edb4639935d71729d
```

Comment:
Since this is a GitHub tarball, the hash should be stable, so @ shouldn't be necessary, why was it added?

Reply:
Actually not sure -- came in via xgensum, so just going to remove it.

Reply:
@ means it's a checksum of the tarball content instead of the tarball itself. It shouldn't be used almost anywhere.

Reply:
I'm aware.